backends/sqldb.go

```diff
@@ -285,7 +286,10 @@ func (s *sqlDB) createTableSchema(cols []string, colTypes []*sql.ColumnType) ins
 		fields[i] = fmt.Sprintf(`"%s" %s`, cols[i], typ)
 	}
 
-	if s.opt.DbType == dbTypePostgres && s.opt.UnloggedTable {
+	// If the DB is Postgres, optionally create an "unlogged" table that disables
+	// WAL, improving performance of throw-away cache tables.
+	// https://www.postgresql.org/docs/9.1/sql-createtable.html
```

Comment:
Here the best is use the `current` link for docs. I mean:

https://www.postgresql.org/docs/current/sql-createtable.html

PS: 9.1 is old and deprecated

Reply:
Woops, fixed: https://github.com/knadh/sql-jobber/commit/b2536b9a614dd35292c81325f7fb02b8c047778d